native/NativeSampleApps/MobileSyncCompose/.gitignore

```diff
@@ -0,0 +1,15 @@
+/build
```

Comment:
You could just add this to the top level `.gitignore` if  you like - I think it's wildcarded anyway.

Reply:
If we have it here, it will show up and the generated app also.

Reply:
Never mind. I just realized the new app is in the main repo. 
If we want it available to customers, we need it to live in SalesforceMobileSDK-Templates.